Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -64,17 +64,12 @@ + (NSDictionary *)appLaunchEnvironmentWithBundleID:(NSString *)hostBundleID
     NSString *testSimulatorFrameworkPath = [[hostAppExecPath stringByDeletingLastPathComponent] stringByDeletingLastPathComponent];
     NSString *dyldLibraryPath = [NSString stringWithFormat:@"%@:%@/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks", testSimulatorFrameworkPath, config.xcodePath];
     NSMutableDictionary<NSString *, NSString *> *environment = [@{
```

Comment:
Thanks for cleaning up all of these, we were cargo-culting these ;-)

Reply:
removed it in the recent commit